pre-commit/README.md

```diff
@@ -0,0 +1,90 @@
+# Pre-commit
```

Comment:
I think it's good to provide info and tools to implement code style checks. However, I think we're going to far and too deep for a first step, and being too intrusive with developer workflows. For example, we're assuming that every developer will be comfortable with managing style via installing webhooks, and are overlooking that some developers will prefer other tools (like having the IDE perform the autoformatting automatically as they write code. Or perform checks as part of the build). 

As a general principle, when we introduce standards I think it's more important that we focus on the goal setting and enforcing a standard, but that we remain *extremely* flexible and respectful with individual developer workflows. We should not assume that every developer works in the same way. And we should not care. What matters is that the standard is respected. Now how.

In practical terms, we should start by just defining the actual standard we want to apply (e.g. google's code style) and provide that spec in the standard way (a codestyle.xml).  This allows any developer / team to integrate The Standard in their workflow or repos, and doesn't impose any assumptions about *how* they do to follow the standard.

THEN we can offer tools that aid them, such as a pre-commit hook. But I think we should not skip the first step.